qa/sc_multiple_pending_certs_non_ceasing.py

```diff
@@ -110,9 +109,9 @@ class SCMultiplePendingCertsNonCeasing(SidechainTestFramework):
 
     sc_nodes_bootstrap_info = None
     sc_withdrawal_epoch_length = 10  # Note: must be even number
-    total_withdrawal_epochs_number = 5
+    total_withdrawal_epochs_number = 5  # sc_withdrawal_epoch_length must exceed this value at least on 2
```

Comment:
Should this read: "...at least **_by_** 2"?

Reply:
Done.